packages/components/Menu/SPEC.md

```diff
@@ -332,6 +332,13 @@ export interface MenuItemProps extends Omit<PressablePropsExtended, 'onPress'> {
    * Do not dismiss the menu when a menu item is clicked
    */
   persistOnClick?: boolean;
+
+  /**
+   * Display the tooltip if one exists
+   * Note the capital "T" in tip as it is needed to match the same prop name in win32 where this is read
+   * @platform win32
+   */
+  alwaysShowToolTip?: boolean;
```

Comment:
My two cents is that we should set this in FURN and not let the client control it. I don't really see a case where the client would set a tooltip and not want it to show, so we should just set alwaysShowTooltip for the client if tooltip is defined. But I think I would talk to PM for clarification

Reply:
I removed the entry from the spec.md file, but should they also get removed from the ***.types.ts files as well?

Reply:
Yeah. We should remove it from the types file, since that would expose the prop